Docs/DesignDoc.md

```diff
@@ -48,6 +48,24 @@ Secret objects supported by this module are currently limited to:
 
 - Hashtable - Hash table of name value pairs, where values are restricted to the above secret types.  
 
+## Secret metadata
+
+Extension vaults can optionally support storing and retrieving additional secret metadata, that is data associated with the secret.  
+Secret metadata is not security sensitive and does not need to be stored securely in the extension vault.  
+
+Secret metadata can be included by using the `-Metadata` parameter: `Secret-Secret -Metadata @{ Name=Value }`.
```

Comment:
do you mean `Set-Secret -Metadata @{Name=Value}`?

Reply:
Yes, thanks!